src/dune/docker.py

```diff
@@ -98,7 +99,10 @@ def rm_file(self, file_name):
         self.execute_cmd(['rm', '-rf', file_name])
 
     def write_file(self, file_name, body):
-        self.execute_cmd(['echo', body, '>', file_name])
+        with tempfile.NamedTemporaryFile(mode='w+',) as tmp:
+            tmp.write(body)
+            tmp.flush()
+            self.cp_from_host(tmp.name, file_name)
```

Comment:
Does Windows allow the copy while tmp remains open? If not, I think we need to add the param `delete=False` and manage close and deletion ourselves.

Reply:
Excellent catch, thanks!
Indeed this is the issue on Windows. Fixed in the latest commit.

Additionally I noticed the tests are not passing on Windows, so I created separate issues for that: https://github.com/AntelopeIO/DUNE/issues/186